diff --git a/score/mw/com/example/com-api-example/com-api-gen/BUILD b/score/mw/com/example/com-api-example/com-api-gen/BUILD
index b855bdf5..9217e658 100644
--- a/score/mw/com/example/com-api-example/com-api-gen/BUILD
+++ b/score/mw/com/example/com-api-example/com-api-gen/BUILD
@@ -21,7 +21,7 @@ rust_library(
     crate_name = "com_api_gen",
     features = ["link_std_cpp_lib"],
     visibility = [
-        "//score/mw/com:__subpackages__",
+        "//visibility:public",
     ],
     deps = [
         ":vehicle_gen_cpp",
